### PR TITLE
Update Docs for code building and add announcement page

### PR DESCRIPTION
**Pull request by spbrogan**

### CODE REVIEW

File: docs/CodeDevelopment/compile.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+It is often desirable to compile test code and at times there might not be a product to test with.  This is also how the Pull Requests gates are implemented and enforced.
+
+!!! Note
+    This also runs the other static code tests so it does more than compile.  
+
+    Assumption is that the repository to compile has been cloned to your filesystem and is in the state ready to compile. 
+
+1. Open cmd prompt at workspace root
+2. Run 'mu_bootstrap` to get the necessary build components
```
Comment (corthon) [inline docs/CodeDevelopment/compile.md]: Is this still accurate? Does it address any module installs that are required to get mu_bootstrap?

File: cspell.json [hunk 2]
```diff
@@ -39,7 +39,7 @@
 
     //Attribution file has lots of external content
```
[the remark this answers is not in the document]
Comment (spbrogan) [reply cspell.json]: file format is controlled by the cspell tool.

File: docs/CodeDevelopment/compile.md (hunk 1 above)
Comment (spbrogan) [reply docs/CodeDevelopment/compile.md]: prereq page has details on current suggested process

File: docs/CodeDevelopment/test.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# Tests
+
+Testing firmware is hard.  Lets just stop there. :poop: If you want to read on please do at your own risk.  Project Mu supports a few types of testing and this page will help provide some high level info and links for more information.
```
[the remark this answers is not in the document]
Comment (out0xb2) [reply docs/CodeDevelopment/test.md]: Test Comment, ignore

[the remark this answers is not in the document]
Comment (spbrogan) [reply docs/CodeDevelopment/test.md]: test comment.  please ignore

File: docs/CodeDevelopment/compile.md (hunk 1 above)
Comment (spbrogan) [reply docs/CodeDevelopment/compile.md]: *Here is another review but this time i used markdown* 

__does this show up as bold__

**Comment by out0xb2** [comment]
TEST: I'm liking the way this is coming together!

**Comment by out0xb2** [comment]
Replying directly from Outlook.com.  In Outlook.com the markdown text does render as bold as expected.


________________________________

**Comment by spbrogan** [comment]
cancel PR.  too much content.  Need to break it up

**Pull request closed without merging**